tutorials/convert_lit_models.md

```diff
@@ -5,34 +5,55 @@ Several models retrieved from external sources must be reformatted with naming c
 The way the script is used depends on the finetuning method. The commands for converting a model back to its original format after using one of the several finetuning methods supported by Lit-GPT are shown below.
 
 > [!NOTE]\
-> each example shown below uses Falcon-7B please be sure to update --checkpoint_dir accordingly
+> all examples use the Meta Llama-2-7b-chat-hf checkpoints and Alpaca dataset during finetuning
+
+> [!NOTE]\
+> finetuned checkpoints will be saved to a subdirectory in a main directory named `out` e.g. finetuning with adapter_v2 will save checkpoints in `out/adapter_v2/alpaca`. that new subdirectory is used as the `--out_dir` argument provided to convert_lit_checkpoint
 
 ### Full Finetuning
 
-After full finetuning, your checkpoint directory will contain a file named `lit_model_finetuned.pth`and converting the finetuned model back to its original weights naming convention can be done by setting the `--checkpoint_name` with:
+After full finetuning, your checkpoint directory will contain a file named `lit_model_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done with the following:
 
 ```sh
 python scripts/convert_lit_checkpoint.py \
-    --checkpoint_name lit_model_finetuned.pth \
-    --checkpoint_dir checkpoints/tiiuae/falcon-7b
+    --checkpoint_name lit_model_finetuned.pth  \
+    --out_dir out/full/alpaca/ \
+    --model_name Llama-2-7b-chat-hf
 ```
 
 ### Adapter and Adapter V2 Finetuning
```

Comment:
As mentioned in other issues/PRs, we don't support adapter exports so this whole section should be removed with an explanation of this limitation

Reply:
updated in [7cbea36](https://github.com/Lightning-AI/lit-gpt/pull/362/commits/7cbea36843faf324908441d6fc08faf86e39259b)